requirements.txt

```diff
@@ -1,5 +1,6 @@
 beautifulsoup4
 fake_useragent
+imageio
```

Comment:
Let's make sure that imageio/imageio#916 is merged.

Reply:
Depending on how you plan to use ImageIO, I'd recommend using `imageio<3.0` or `imageio<4.0` instead. We will release v3.0 around the end of January which will remove deprecations. Pinning up to the next version major avoids your code breaking once that happens and puts you in charge of when/how to upgrade.

Reply:
We are a bit different than most repos.  In general, we do not pin dependencies because we are not shipping an executable (app or library) and our individual algorithms are almost always self-standing.  We hope that tests catch incompatibilities so that they can be fixed as they break.  If not then users can report failures so we fix-as-we-go.